Docs: Plugin E2E - Using the api article (#885)
grafana/plugin-tools: docusaurus/docs/e2e-test-a-plugin/get-started.md

queryString="current-package-manager"
/>

### Step 3: Run tests in CI

> The [`grafanaDependency`](../metadata.md#properties-1) property in the plugin.json file dictates what versions of Grafana the plugin is compatible with. It's recommended to run your Playwright end-to-end tests targeting all the supported versions. The Github workflow that was included when scaffolding the plugin ensures this is the case.

Reviewer: ```suggestion
The [`grafanaDependency`](../metadata.md#properties-1) property in the `plugin.json` file specifies what versions of Grafana the plugin is compatible with. As a best practice, run your Playwright end-to-end tests targeting all the supported versions. The GitHub workflow that was included when scaffolding the plugin ensures this is the case.
```

> The [`grafanaDependency`](../metadata.md#properties-1) property in the `plugin.json` file specifies what versions of Grafana the plugin is compatible with. As a best practice, run your Playwright end-to-end tests targeting all the supported versions. The GitHub workflow that was included when scaffolding the plugin ensures this is the case.

Reviewer 2: ```suggestion
The [`grafanaDependency`](../metadata.md#properties-1) property in the `plugin.json` file specifies what versions of Grafana the plugin is compatible with. As a best practice, run your Playwright end-to-end tests targeting all the supported versions. The GitHub workflow that can be included when scaffolding plugins with `create-plugin` ensures this is the case.
```

> If you chose to not add a Github workflow when you scaffolded the plugin, it's recommended to follow the instructions in the [CI](./ci.md) guide to run Playwright end-to-end tests targeting all versions of Grafana that your plugin supports.

Reviewer: ```suggestion
If you chose to not add a GitHub workflow when you scaffolded the plugin, as a best practice follow the instructions in the [CI](./ci.md) guide to run Playwright end-to-end tests targeting all versions of Grafana that your plugin supports.
```

> 

Author: Even though unrelated to the new API article, I'm taking the chance to update this section as it was outdated.
